osu.Game/Screens/Select/BeatmapDetails.cs

```diff
@@ -381,6 +373,12 @@ public Color4 TextColour
                 get { return textFlow.Colour; }
                 set { textFlow.Colour = value; }
             }
+
+            [BackgroundDependencyLoader]
+            private void load()
+            {
+                textFlow.Colour = Color4.White.Opacity(0.75f);
```

Comment:
This doesn't need to be done in BDL anymore and can be set when creating `textFlow`.